docs/software/spack.md

```diff
@@ -0,0 +1,179 @@
+
+### Activate and Use the CURC Spack Module
+
+Run the following command to load the Spack module:  
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ module load spack/0.20.1
+```
+
+You can confirm that spack has been loaded and find a list of useful spack commands by running the `spack` command: 
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack
+```
+
+### Using Spack Environments
+
+Now that you've loaded the Spack module, you'll need to create a spack environment in which you'll install software.
+
+__1. Create a Spack environment__
+You can create a Spack environment with the following command: 
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack env create <environment name>
+```
+
+Note that, by default, environment specs are stored in `/projects/$USER/spack/environments/`. This location can be changed by modifying the `environments_root` variable within `~/.spack/config.yaml`.
+
+__2. Activate your Spack environment__
+You can activate your Spack environment with one of the following commands: 
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spacktivate <environment name>
+```
+
+or
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack env activate <environment name>
+```
+
+To deactivate your environment, you can use one of the following commands: 
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ despacktivate
+```
+
+or
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack env deactivate
+```
+
+### Installing Software with Spack
+
+Once you've created and activated an environment, you can begin installing software using Spack. To see a list of *all* of the software available, you can use the spack list command: 
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack list
+```
+
+The above command lists all 7000+ packages available for installation with Spack. As a result, this command may take some time to execute.  To search for a specific software by name, you can use `spack list <software name>`. For example: 
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack list fastqc
+fastqc
+==> 1 packages
+```
+
+>For additional information about a specific piece of software (available versions, variants, and dependencies), you can use the `spack info <software name>` command. Note that the 'preferred version' is the default version which will be installed if no alternative version is specified. 
+
+Once you've confirmed that a piece of software is available, you can install it in the active environment using the `spack install --add` command:
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack install --add fastqc
+```
+>Note that the `--add` argument is required to add package specs to an environment. You can separately add specs without installing a package using the `spack add <software name>` command.
+
+This will install the specified software and all dependencies into your environment.
+
+You can also modify the installation commands to meet your installation needs. You can specify which version you'd like installed using the `@` operator:
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack install --add fastqc@0.11.9
+```
+
```

Comment:
```suggestion
Once you have installed a package, you can view what packages are installed in your environment using the `spack find` command. For example, 
```code
[johndoe@c3cpu-c11-u17-2 ~]$ spack find
==> In environment my_test_environment
==> Root specs
fastqc@0.11.9

==> Installed packages
-- linux-rhel8-zen / gcc@8.5.0 ----------------------------------
berkeley-db@18.1.40  fastqc@0.11.9  ncurses@6.4        pkgconf@1.9.5
bzip2@1.0.8          gdbm@1.23      openjdk@11.0.17_8  readline@8.2
diffutils@3.9        libiconv@1.17  perl@5.36.0        zlib@1.2.13
==> 12 installed packages
```code
>Note that since we did not specify a compiler, the default compiler of `gcc@8.5.0` was used. Please also note that the default compiler may change. 

```

I am not exactly sure "```code" is the correct formatting, I had to do that because of the GitHub suggestions. Please double check the formatting.